components/gfx/font_cache_thread.rs

```diff
@@ -185,6 +185,7 @@ impl FontCache {
                                               metadata.content_type);
                                         *response_valid.lock().unwrap() = is_response_valid;
                                     }
+                                    ResponseAction::HeadersAvailable(Err(_)) => {}
```

Comment:
This should really be dealt with by the previous block so we get proper logging.